### PR TITLE
Feature/user crud

### PR DESCRIPTION
**Pull request by Hemi102**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+module Admin
+  class UsersController < ApplicationController
+    def index
+      @users = User.all
```
Comment (Sohair63) [inline app/controllers/admin/users_controller.rb:4]: do not use all, maybe remove once you have pagination

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    end
+
+    def show
+      @user = User.find(params[:id])
+    end
+
+    def edit
+      @user = User.find(params[:id])
```
Comment (Sohair63) [inline app/controllers/admin/users_controller.rb:12]: try sending random ID from browser and see what happend.

File: app/controllers/admin/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    end
+
+    def new
+      @user = User.new
+    end
+
+    def create
+      @user = User.new(user_params.merge(confirmed_at: DateTime.now))
+      pass = SecureRandom.base64(10)
+      @user.password = pass
```
Comment (Sohair63) [inline app/controllers/admin/users_controller.rb:31]: business logic should be in model or service

File: app/controllers/admin/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      if @user.save
+        UserMailer.send_password(@user, pass).deliver_now
+        redirect_to admin_users_path
+      else
+        render 'new'
+      end
+    end
+
+    def destroy
+      User.find(params[:id]).destroy
```
Comment (Sohair63) [inline app/controllers/admin/users_controller.rb:41]: why not using before action here?

File: app/controllers/admin/users_controller.rb (hunk 1 above)
Comment (Hemi102) [reply app/controllers/admin/users_controller.rb:5]: yes sir this will removed when pagination applied

**Pull request merged**